training-maquetado/scss/detail.scss

```diff
@@ -68,3 +68,51 @@
     width: 100%;
   }
 }
+
+@media only screen and (max-width: $desktop-min-width) {
+  .book {
+    flex-direction: column;
+    height: 570px;
+    max-width: 336px;
+    width: 100%;
+    margin: 0 0 20px 0;
+    padding-top: 38px;
+  }
+
+  .book-cover {
+    align-self: center;
+    margin: 0;
+    margin-bottom: 21px;
+
+    &::before {
+      background-size: calc(95px + -30%) calc(120px + -30%);
```

Comment:
this is weird :/ why?

Reply:
https://trello.com/c/LPLWeDS6/34-responsive the training card says that the badge is 30% smaller in mobile resolution and that i could use calc for that, also sice the badge it's a pseudo-element changing the width had no effect in the badge size so I used background-size. Is there a cleaner way to do this?

Reply:
the width should change it doesn't matter if its a pseudo element. you should set width and height. Also you can just do calc($width - 30%) no need for the + and -